config/gocdb_schema.xml

```diff
@@ -266,6 +266,9 @@
 	    <length>255</length>
 	    <regex>/^(\+?[0-9\-\.\s\)\(]+)?$/</regex>
 	</field>
+	<field>
+		<fname>UNLINK_HOMESITE</fname>
```

Comment:
This line seems overly indented (1 tab instead of 4 spaces?).

The below might fix it, it's hard to tell in GitHub though...

```suggestion
        <fname>UNLINK_HOMESITE</fname>
```